quarkus/tests/integration/src/test/java/org/keycloak/it/cli/dist/QuarkusPropertiesAutoBuildDistTest.java

```diff
@@ -52,6 +54,7 @@ void reAugOnFirstRun(LaunchResult result) {
     @BeforeStartDistribution(EnableAdditionalConsoleHandler.class)
     @Launch({ "start" })
     @Order(2)
+    @Disabled(value = "We don't properly differentiate between quarkus runtime and build time properties")
```

Comment:
Instead of disabling all of them, it'd be good to have some test, that the reaug really happening, when the quarkus prop is changed. @shawkins WDYT?

Reply:
We do have that for the build-time quarkus properties, the disabled tests should be for runtime only.